lib/remote.js

```diff
@@ -3,44 +3,61 @@
 const path = require('path');
 const os = require('os');
 const fs = require('fs-extra');
+const unzip = require('unzip2');
 const config  = require('./config');
+const utils = require('./utils');
 
-// Downloads the zip file from github and extracts it to /tmp/tldr
+let request = require('request');
+
+function createTempFolder(prefix = 'tldr') {
+  let tmpdir = path.join(os.tmpdir(), prefix);
+  let target = path.join(tmpdir, utils.uniqueId());
+
+  return fs.mkdirp(target).then(() => {
+    return {
+      get path() {
```

Comment:
It is not immediately apparent why we are returning an object like this. Add a comment ?